test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreCommand/OriginalCaseGlobalPackageFolderTests.cs

```diff
@@ -23,6 +23,8 @@
 
 namespace NuGet.Commands.Test
 {
+    using LocalPackageArchiveDownloader = NuGet.Protocol.LocalPackageArchiveDownloader;
```

Comment:
The using statements sit outside of the namespace.

Reply:
Seeing your change below...is there an ordering issue?

Reply:
In one of the 3 test files, it didn't work for some reason. Since this is defining an alias for a specific type, rather than making a namespace available for use, I thought it isn't so important to keep it with the other usings.